### PR TITLE
Refactor Errors E2E test to no longer rely upon a real error

### PR DESCRIPTION
**Pull request by brad-decker**
## Explanation
We were previously relying on a known bug to test our integration with Sentry (making sure sentry attempts to send errors to the server). This PR replaces that by triggering an error that will only exist in Test.

## Manual Testing Steps
1. Build test mode
2. In the console, type 'stateHooks.throwTestError()'
3. View the network tab and see that it was sent to sentry (or would have been)
4. Also check E2E test works.

## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved
  - [x] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [x] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

File: test/e2e/tests/errors.spec.js [hunk 1]
```diff
@@ -43,12 +43,13 @@ describe('Sentry errors', function () {
         await driver.fill('#password', 'correct horse battery staple');
         await driver.press('#password', driver.Key.ENTER);
         // Trigger error
-        await driver.clickElement('[data-testid="eth-overview-send"]');
-        await driver.fill(
-          'input[placeholder="Search, public address (0x), or ENS"]',
-          '0x2f318C334780961FB129D2a6c30D0763d9a5C970',
-        );
-        await driver.fill('input[placeholder="0"]', `-01`);
+        // await driver.clickElement('[data-testid="eth-overview-send"]');
```
Comment (DDDDDanica) [inline test/e2e/tests/errors.spec.js:46]: They can be removed ?

Comment (brad-decker) [reply test/e2e/tests/errors.spec.js:46]: Oh yeah good point lol

Comment (brad-decker) [reply test/e2e/tests/errors.spec.js]: comments removed

**Review by pedronfigueiredo — APPROVED** [review]

**Review by DDDDDanica — APPROVED** [review]

File: test/e2e/tests/errors.spec.js [hunk 2]
```diff
@@ -43,12 +43,7 @@ describe('Sentry errors', function () {
         await driver.fill('#password', 'correct horse battery staple');
         await driver.press('#password', driver.Key.ENTER);
         // Trigger error
-        await driver.clickElement('[data-testid="eth-overview-send"]');
-        await driver.fill(
-          'input[placeholder="Search, public address (0x), or ENS"]',
-          '0x2f318C334780961FB129D2a6c30D0763d9a5C970',
-        );
-        await driver.fill('input[placeholder="0"]', `-01`);
+        driver.executeScript('window.stateHooks.throwTestError()');
```
Comment (PeterYinusa) [inline test/e2e/tests/errors.spec.js:46]: nice!

**Review by PeterYinusa — APPROVED** [review]

**Pull request merged**